src/Styleguide/Components/index.tsx

```diff
@@ -0,0 +1,22 @@
+export * from "./ArtistBio"
+export * from "./ArtistCard"
+export * from "./ArtworkGrid"
+export * from "./ArtworkGridExample"
+export * from "./AuctionCard"
+export * from "./Collapse"
+export * from "./CountrySelect"
+export * from "./FollowIcon"
+export * from "./Footer"
+export * from "./MarketInsights"
+export * from "./MobileTopBar"
+export * from "./Pagination"
+export * from "./ReadMore"
+export * from "./RecentlyViewed"
+export * from "./RouteTabs"
+export * from "./SelectedExhibitions"
+export * from "./Slider"
+export * from "./Stepper"
+export * from "./StepSummaryItem"
+export * from "./Tabs"
+export * from "./Toggle"
+export * from "./Lightbox"
```

Comment:
I wonder if there's a lint rule that exists to ensure every file is included in this export list...